ethereum/core/src/main/java/org/hyperledger/besu/ethereum/trie/diffbased/verkle/worldview/VerkleWorldState.java

```diff
@@ -453,7 +450,9 @@ public Hash frontierRootHash() {
     return calculateRootHash(
         Optional.of(
             new VerkleWorldStateKeyValueStorage.Updater(
-                noOpSegmentedTx, noOpTx, worldStateKeyValueStorage.getFlatDbStrategy())),
+                noOpSegmentedTx,
+                noOpTx,
+                (VerkleStemFlatDbStrategy) worldStateKeyValueStorage.getFlatDbStrategy())),
```

Comment:
Do you need to define `frontierRootHash`? From what I can see this is only used to define transaction receipts compliant with Frontier but Berlin is already overriding that. I can't see why we need to have verkle with Frontier.

Reply:
I prefer to keep it this way for now because I’d like to sync the mainnet with Verkle for testing purposes, and I might need this method. We can remove it later when it’s no longer useful.

Reply:
Maybe add a comment that you want it for that?